sdk/core/System.Net.ClientModel/tests/System.Net.ClientModel.Tests.csproj

```diff
@@ -1,19 +1,27 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
+		<Description>Tests of System.Net.ClientModel types and public contracts.</Description>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <IsTestProject>true</IsTestProject>
+		<Nullable>enabled</Nullable>
```

Comment:
We need nullability enabled for the new library so that we can properly test API contracts.